run_dir/design/project_samples.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+python {% extends "base.html" %}
```

Comment:
Hah - nothing, I suspect it prints the word `python` at the top of the html. I guess that this is a typo from when my windows shifted focus, and I had meant to type that into a terminal window. Good spot ;)